base/sysinfo.jl

```diff
@@ -323,6 +323,22 @@ windows_version
 
 const WINDOWS_VISTA_VER = v"6.0"
 
+"""
+    Sys._can_execute(path::AbstractString)
+
+Returns `true` if the given `path` has executable permissions.
+"""
+function _can_execute(path::AbstractString)
+    # We use `access()` and `X_OK` to determine if a given path is executable
+    # by the current user.  `X_OK` comes from `unistd.h`.
+    X_OK = 1 << 0
+    @static if iswindows()
+        ccall(:_access, Cint, (Ptr{UInt8}, Cint), path, X_OK) == 0
```

Comment:
`access` on windows doesn't have an `X_OK` parameter

Reply:
Ah, that's an excellent point.  I just spent half an hour trying to parse MSDN and figure out how `GetNamedSecurityInfo` can help me figure out the right ACL values to see if something is executable by the current user, but it defeated me.  I'm just going to return `true` on Windows.

Reply:
Yes, the `which` function on Windows probably shares fairly little with the same function on posix. I expect they likely should just be completely independent implementations

Reply:
.........`busybox-w32` uses `access(path, X_OK)`.  Sigh.

Reply:
I’m not too surprised. It’s a posix emulation later, probably intended to approximate the result you’d get on `busybox sh` or cygwin (similar to how the real which program on posix approximates the likely answer)